my_list.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] You did a great job with the `MyEnumerable` module. :+1: If you see the code used to verify the solution, our list should be able to take an arbitrary number of arguments while the `MyList` class you created is accepting an array. You can use `*args` as a parameter so your class can accept multiple arguments. Then you can loop over the arguments and add them to your `@list` instance variable. Please update it accordingly. An example implementation is as follows. ✍️

```
def initialize(*args)
    @list = []

    args.each { |arg| @list << arg }
end
```